mpicbg/src/main/java/mpicbg/ij/InteractiveInvertibleCoordinateTransform.java

```diff
@@ -75,6 +75,9 @@ final static protected void useRoi( final float[] x, final float[] y )
 	abstract protected void setHandles();
 	abstract protected void updateHandles( int x, int y );
 	
+	// Convenience method to return something after ENTER is pressed.
```

Comment:
How about

    protected void onReturn(){}

then, implementing classes would not need to override it if they don't like it.